demo/src/main/java/com/bluelinelabs/conductor/demo/changehandler/transitions/FabTransform.java

```diff
@@ -274,30 +254,34 @@ public void getOutline(View view, Outline outline) {
         colorFade.setInterpolator(fastOutSlowInInterpolator);
         iconFade.setInterpolator(fastOutSlowInInterpolator);
 
-        // Work around issue with elevation shadows. At the end of the return transition the shared
-        // element's shadow is drawn twice (by each activity) which is jarring. This workaround
-        // still causes the shadow to snap, but it's better than seeing it double drawn.
-        Animator elevation = null;
-        if (!fromFab) {
-            elevation = ObjectAnimator.ofFloat(view, View.TRANSLATION_Z, -view.getElevation());
-            elevation.setDuration(duration);
-            elevation.setInterpolator(fastOutSlowInInterpolator);
-        }
-
         // Run all animations together
         final AnimatorSet transition = new AnimatorSet();
         transition.playTogether(circularReveal, translate, colorFade, iconFade);
         transition.playTogether(fadeContents);
-        if (elevation != null) transition.play(elevation);
-        if (fromFab) {
-            transition.addListener(new AnimatorListenerAdapter() {
-                @Override
-                public void onAnimationEnd(Animator animation) {
-                    // Clean up
-                    view.getOverlay().clear();
-                }
-            });
+        if (dialogView != null) {
+            final Animator dialogViewFade = ObjectAnimator.ofInt(dialogView, "alpha", 0).setDuration(twoThirdsDuration);
+            dialogViewFade.setInterpolator(fastOutSlowInInterpolator);
+            transition.playTogether(dialogViewFade);
         }
+        transition.addListener(new AnimatorListenerAdapter() {
+            @Override
+            public void onAnimationEnd(Animator animation) {
+                // Clean up
+                view.getOverlay().clear();
+
+                if (!fromFab) {
+                    view.setTranslationX(0);
+                    view.setTranslationY(0);
+                    view.setTranslationZ(0);
+
+                    view.measure(
+                            makeMeasureSpec(196, View.MeasureSpec.EXACTLY),
```

Comment:
Wouldn't it be better here to retrieve size from captureValues? Or expect it as a constructor argument?
Instead of hard coding it?